x-pack/plugins/triggers_actions_ui/public/application/lib/rule_api/map_filters_to_kql.test.ts

```diff
@@ -32,14 +32,40 @@ describe('mapFiltersToKql', () => {
     ]);
   });
 
-  test('should handle ruleStatusesFilter', () => {
+  test('should handle ruleExecutionStatusesFilter', () => {
     expect(
       mapFiltersToKql({
-        ruleStatusesFilter: ['alert', 'statuses', 'filter'],
+        ruleExecutionStatusesFilter: ['alert', 'statuses', 'filter'],
       })
     ).toEqual(['alert.attributes.executionStatus.status:(alert or statuses or filter)']);
   });
 
+  test('should handle ruleStatusesFilter', () => {
+    expect(
```

Comment:
should we add a test for just `disabled` too?

Reply:
yep, good idea!